MetaMask/metamask-mobile
# Integrating ppom update tool into mobile build

## **Description**
Add ppom-view tool https://github.com/MetaMask/ppom-view. into metamask mobile to add generation `ppom.html.js` generation into mobile build.

## **Manual testing steps**

_1. Run mobile build
_2. Build should run smoothly
_3. Build should generate new `ppom.html.js`

## **Related issues**

_Fixes: MetaMask/MetaMask-planning#1362

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
